entity-definitions/src/main/java/eu/europeana/entity/definitions/model/vocabulary/WebEntityFields.java

```diff
@@ -23,6 +23,9 @@ public interface WebEntityFields {
     public static final String THUMBNAIL = "thumbnail";
     public static final String IS_SHOWN_BY = "isShownBy";
 
+    //Enrichment field
+    public static final String ENRICH_LABEL_FIELD = "label";
```

Comment:
shouldn't this be label_enrich?

Reply:
oh yes, i think for testing i changed it and for got to revert it.